plenum/server/node.py

```diff
@@ -2771,7 +2771,7 @@ def verifySignature(self, msg):
             return
         if isinstance(msg, Propagate):
             typ = 'propagate'
-            req = msg.request
+            req = self.client_request_class(**msg.request)
```

Comment:
This is done twice, the second one is in the processPropagate() function. It means that sha256 (the digest) is calculated twice too that affects the performance. Can we get rid of it?